.github/workflows/ci.yml

```diff
@@ -31,21 +31,57 @@ jobs:
             cxx: clang++
             type: static
             backend: JavaScriptCore
+            javascriptcore_version: "4.0"
+            options: -DINCLUDEJS_BACKEND_JAVASCRIPTCORE_API_VERSION:STRING=4.0
           - os: ubuntu-latest
             cc: gcc
             cxx: g++
             type: static
             backend: JavaScriptCore
+            javascriptcore_version: "4.0"
+            options: -DINCLUDEJS_BACKEND_JAVASCRIPTCORE_API_VERSION:STRING=4.0
```

Comment:
I wonder if we could not use javascriptcore_version to set `DINCLUDEJS_BACKEND_JAVASCRIPTCORE_API_VERSION` later.

Reply:
see my latest commit, we could do it like that... but then it ends up being specified (as empty) for non-linux ci runs

Reply:
i mean it being empty on non-linux isn't really an issue... since the `FindJavaScriptCore.cmake` just ignores this on macos anyway. although we might want to check that it's not set and warn otherwise

Reply:
any idea why the asan would fail here? i think nothing should have changed

Reply:
Hey I already experienced weird stuff with ASAN: https://bugs.webkit.org/show_bug.cgi?id=269791